packages/cfd/src/Containers/mt5-mobile-redirect-option.scss

```diff
@@ -0,0 +1,39 @@
+.mt5-download-container {
+    display: flex;
+    flex-direction: column;
+    gap: 1.6rem;
+    margin: 2.4rem;
+
+    &--span {
+        width: 100%;
+    }
+
+    &--option {
+        width: 100%;
+        height: 3.4rem;
+        padding: 0.8rem;
+        border-radius: 0.8rem;
+        background-color: var(--general-1);
+        .center {
+            gap: 0.8rem;
+            width: 100%;
+            display: flex;
+            justify-content: space-between;
+            p {
+                flex: 1;
+            }
```

Comment:
it will be nicer to have some sort of classNames for these. It will be easy to identify. 🙂